build.rs

```diff
@@ -0,0 +1,39 @@
+fn main() {
+    let out_dir = std::env::var("OUT_DIR").unwrap();
+    let target = std::env::var("TARGET").unwrap();
+
+    let machine = if target.contains("x86_64") {
+        "X64"
+    } else if target.contains("ARM64") {
+        "ARM64"
+    } else if target.contains("ARM") {
+        "ARM"
+    } else if target.contains("i686") {
+        "X86"
+    } else {
+        panic!("Unsupported target");
+    };
+
+    let lib = cc::windows_registry::find_tool(&target, "lib.exe").expect("lib.exe not found");
+
+    for path in std::fs::read_dir("def")
+        .unwrap()
```

Comment:
Let's use expect here. `def` "folder has been deleted or cannot be read for some reason".